src/index.html

```diff
@@ -7,8 +7,49 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__left-side">
+        <a href="#home" class="nav__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <div class="header__right-side">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
```

Comment:
I suggest you putting 1 blank line between sibling tags (when they have some content) to improve readability